unime/src/routes/+layout.svelte

```diff
@@ -106,8 +183,11 @@
 
   <!-- Dev Mode: Debug messages -->
   {#if showDebugMessages}
-    <div class="relative z-10 min-h-full w-screen bg-orange-100">
-      <p class="p-4 text-center text-xs font-semibold uppercase text-orange-800">debug messages</p>
+    <div class="relative z-10 min-h-full w-screen bg-orange-100 pt-24">
+      <p class="pb-2 pt-2 text-center text-xs font-semibold uppercase text-orange-800">debug messages</p>
+
+      <hr class="mx-8 h-0.5 border-t-0 bg-orange-800 opacity-100" />
```

Comment:
* Why set `opacity: 100%` here explicitly? Did you encounter an issue when not setting it?
* Why `border-top-width: 0px`? If you want the divider to be thicker, why not use a different `height`?

Reply:
Opacity I removed, border-top-width looks better than the values tailwind give me 0.5 it too little h-1 is too big

Reply:
If you feel like the presets are not suitable, you can pass any value in brackets such as: `h-[3px]`. See the documentation on [using arbitrary values](https://tailwindcss.com/docs/height#arbitrary-values). I don't think setting the border-width to 0 pixels just because it visually fixes the problem is sustainable. The next person to work on this code will have questions.